README.md

```diff
@@ -183,3 +170,41 @@ https://github.com/axios/axios
 Ok thats the basics! You are ready to start playing with the application. We will sit together when youve got the basics set up here and brainstorm some ideas for what this application might do. When you are ready to commit some code I will make you a "collaborator" on this Github repository and then you can commit and push your branch, creating a pull request that I can review and subsequently merge into the master branch. We will go over the details of that when you are ready to do so.
 
 Have fun!
+
+### Additional reading
```

Comment:
moved this section about configuration files down a bit - just trying to reduce the shear amount of tools/terminology and make it clear they can save this deep dive for later